app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_line = command.split(" ")
+    command, file_1, file_2 = command_line
```

Comment:
You need to check that command_line have exactly 3 params

Reply:
also file_1 and file_2 is not clear names
file_1 -> source_path
file_2 -> copy_path